src/components/Comments.js

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../utils/api";
+import "../styles/Comments.css";
+
+const Comments = ({ review_id }) => {
+  const [allComments, setAllComments] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [moreComments, setMoreComments] = useState(false);
+
+  const handleMoreCommentClick = () => {
+    moreComments === true ? setMoreComments(false) : setMoreComments(true);
+  };
+
+  useEffect(() => {
+    setLoading(true);
```

Comment:
Do you need this setLoading(true) here? You're already setting it true on line 7 when the state is created.